cftime/_cftime.pyx

```diff
@@ -1533,12 +1533,17 @@ Gregorial calendar.
         return '%Y-%m-%d %H:%M:%S'
 
     def strftime(self, format=None):
+        """
+        Return a string representing the date, controlled by an explicit format
+        string. For a complete list of formatting directives, see section
+        strftime() and strptime() Behavior in the base Python documentation.
```

Comment:
nit: behavior should be lower case

Reply:
Actually it is the title of a section of documentation and should be uppercase. Also italicized but that's not possible. Maybe quotation marks?

Reply:
Fixed.